rules/TMPRL1102.md

```diff
@@ -0,0 +1,60 @@
+# TMPRL1102 - Workflow finished while handlers are still running.
+
+## Cause
+
+A signal or update handler had not finished executing when the workflow finished (completed, or continued-as-new).
+
+**Note: this warning and the following discussion do not apply to signals in Go.**
```

Comment:
I get the feeling this kind of "does not apply to" warnings will be very common. Maybe we should add some kind of "Applies to:" metadata at the top of the file, to formalize that data point.

CC @cretz: WDYT?